app/assets/javascripts/game.js

```diff
@@ -0,0 +1,77 @@
+$(function() {
+    // Picks up all the latest positions from
+    // the database
+    var currentPositions = $('.piece_pos').data('positions');
```

Comment:
May want to use an ID instead of a class selector here in order to guarantee a unique result. There can be two of the same class, but only one Id. jQuery selects anything that matches, so multiple matches would really screw with the intent.